module/zstd/zfs_zstd.c

```diff
@@ -50,6 +50,10 @@
 #include "lib/zstd.h"
 #include "lib/common/zstd_errors.h"
 
+int zstd_lz4_pass = 1;
+int zstd_firstpass_level = ZIO_ZSTD_LEVEL_1;
+unsigned int zstd_abort_size = 131072;
```

Comment:
`static`?

Reply:
Thought I wrote a reply to this, but "sure, for any of the ones that remain - I was at least planning to drop the first one unless people feel strongly about it, and probably make the second not tunable."